daphne_worker/src/durable/mod.rs

```diff
@@ -35,14 +36,27 @@ const ERR_NO_VALUE: &str = "No such value in storage.";
 // TODO(bhalley) does this need to be configurable?
 const MAX_KEYS: usize = 128;
 
+const DEFAULT_RETRY_INTERVAL: Duration = Duration::from_millis(500);
+const DEFAULT_MAX_RETRIES: usize = 3;
```

Comment:
Reviwer note: Eventually we'll want these parameters to be overwritable by the wrangler.toml. However for now I think we should err on the side of simplicity while we figure out what we actually want in a retry mechanism long-term.